packages/transport/src/modules/relationshipTemplates/RelationshipTemplateController.ts

```diff
@@ -112,8 +112,10 @@ export class RelationshipTemplateController extends TransportController {
     }
 
     public async deleteRelationshipTemplate(template: RelationshipTemplate): Promise<void> {
-        const response = await this.client.deleteRelationshipTemplate(template.id.toString());
-        if (response.isError) throw response.error;
+        if (template.isOwn) {
+            const response = await this.client.deleteRelationshipTemplate(template.id.toString());
+            if (response.isError) throw response.error;
+        }
```

Comment:
make sure to catch these errors during decomposition, we shouldn't stop the decomposition b/c of an error occurs here

Reply:
even better :)